README.md

```diff
@@ -9,15 +9,16 @@ You can find more resources on Safe Apps in the [Gnosis Safe Developer Portal](h
-| [safe-apps-onboard](https://github.com/gnosis/safe-apps-sdk/tree/master/packages/safe-apps-onboard)                                                                                                                          | A wrapper around Onboard.js that would automatically connect to the Safe if the app is loaded as a Safe app           |
-| [safe-apps-web3modal](https://github.com/gnosis/safe-apps-sdk/tree/master/packages/safe-apps-web3modal)                                                                                                                          | A wrapper around Web3modal that would automatically connect to the Safe if the app is loaded as a Safe app                         |
+| Package                                                                                                                                                                                                                        | Description                                                                                                 |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :---------------------------------------------------------------------------------------------------------- |
+| [cra-template-safe-app](https://github.com/gnosis/safe-apps-sdk/tree/master/packages/cra-template-safe-app)                                                                                                                    | CRA template to quickly bootstrap a Safe app                                                                |
+| [safe-apps-react-sdk](https://github.com/gnosis/safe-apps-sdk/tree/master/packages/safe-apps-react-sdk)                                                                                                                        | A wrapper of safe-apps-sdk with helpful hooks                                                               |
+| [safe-apps-sdk](https://github.com/gnosis/safe-apps-sdk/tree/master/packages/safe-apps-sdk) [![npm version](https://badge.fury.io/js/%40gnosis.pm%2Fsafe-apps-sdk.svg)](https://badge.fury.io/js/%40gnosis.pm%2Fsafe-apps-sdk) | JavaScript SDK                                                                                              |
+| [safe-apps-ethers-provider](https://github.com/gnosis/safe-apps-sdk/tree/master/packages/safe-apps-ethers-provider)                                                                                                            | An ethers.js provider and signer to be used with ethers.js contract instances                               |
```

Comment:
Should we remove that one?

Reply:
We should, I thought I'd do it in a separate PR so its easier to review